### PR TITLE
Fix various I18N errors (2)

### PR DESCRIPTION
**Pull request by Ruin0x11**
# Related Issues

#715.

### CODE REVIEW

File: src/elonacore.cpp [hunk 1]
```diff
@@ -6403,7 +6403,7 @@ void initialize_adata()
     p = mdata_t::map_id_t::test_world;
     adata(16, p) = p;
     adata(15, p) = 0;
-    adata(0, p) = mdata_t::map_type_t::world_map;
+    adata(0, p) = mdata_t::map_type_t::field;
```
Comment (ki-foobar) [inline src/elonacore.cpp]: I don't understand why you changed this value. The previous value `world_map` is correct, I think.

File: runtime/locale/jp/damage.hcl [hunk 2]
```diff
@@ -35,8 +35,8 @@ locale {
             attacks_with = "${kare_wa(_1)}${name(_3)}に${_4}で${_2}。"
             attacks_and = "${kare_wa(_1)}${name(_3)}を${_2}${_4}"
 
-            attacks_unarmed = "${kare_wa(_1)}${name(_3)}に${_2}"
-            attacks_unarmed_and = "${kare_wa(_1)}${name(_3)}を${_2}${_4}"
+            attacks_unarmed = "${kare_wa(_3)}${name(_1)}に${_2}"
+            attacks_unarmed_and = "${kare_wa(_3)}${name(_1)}を${_2}${_4}"
```
Comment (ki-foobar) [inline runtime/locale/jp/damage.hcl]: You don't need to change `attacks_unarmed_and`.

The previous one is right.
```hcl
attacks_unarmed_and = "${kare_wa(_1)}${name(_3)}を${_2}${_4}"
```

File: src/elonacore.cpp (hunk 1 above)
Comment (Ruin0x11) [reply src/elonacore.cpp]: Yes, might have been an extraneous change.

**Comment by ki-foobar** [comment]
dropped some unnecessary commits and merged.

**Pull request closed without merging**